package/psyneulinkviewer/conda.py

```diff
@@ -0,0 +1,73 @@
+import platform
+import os
+import sys
+import subprocess
+import logging
+import configuration
+import wget
+import platform
+from setuptools import setup, find_packages
+from setuptools.command.install import install
+from packaging.version import Version
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def activate_env():
+    logging.info("Creating conda ")
+    subprocess.run(configuration.create_env, shell=True)
+
+    logging.info("Activating conda ")
+    subprocess.run(configuration.activate_env, shell=True)
+
+def install_conda():
+    if os.name == 'posix':
+        bash_file = wget.download(configuration.linux_conda_bash, out="psyneulinkviewer")
+    elif platform.system() == 'Darwin':
+        bash_file = wget.download(configuration.mac_conda_bashf)
+
+    logging.info("Installing conda %s", bash_file)
+    logging.info(bash_file)
+    subprocess.run("chmod +x " + bash_file, shell=True)
+    subprocess.run(bash_file + " -b -u -p ~/miniconda3", shell=True)
+
+    activate_env()
+    logging.info("Clean up ")
+    subprocess.run("rm -rf " + bash_file, shell=True)
+
+
+def check_conda_installation():
+    conda_version = subprocess.run(
+        ["conda", "--version"],
+        capture_output = True,
+        text = True 
+    ).stdout
+    if conda_version:
+        conda_version = conda_version.split(" ")[1]
+    logging.info("Conda version detected : %s", conda_version)
+
+    if conda_version is not None:
```

Comment:
should not this be if conda_version is None?

Reply:
True, it's a typo left over form testing, forcing it to fail